content/languages/ruby/authoring.md

```diff
@@ -0,0 +1,156 @@
+---
+kind: tutorial
+languages: [ruby]
+sidebar: "language:ruby"
+prev: /languages/ruby/codewars-test/
+---
+
+# Ruby: creating and translating a kata
+
+This article is intended for kata authors and translators who would like to create new content in JavaScript. It attempts to explain how to create and organize things in a way conforming to [authoring guidelines](/authoring/guidelines/), shows the most common pitfalls, and how to avoid them.
```

Comment:
"in JavaScript"?

Reply:
```suggestion
This article is intended for kata authors and translators who would like to create new content in Ruby. It attempts to explain how to create and organize things in a way conforming to [authoring guidelines](/authoring/guidelines/), shows the most common pitfalls, and how to avoid them.
```